js/data/locale/ky/dateformats.json

```diff
@@ -3,40 +3,40 @@
         "order": "{date} {time}",
         "date": {
             "dmwy": {
-                "f": "‏yyyy-'ж'., d-MMMM, EEEE",
-                "l": "‏yyyy-'ж'., d-MMMM, EEE",
-                "m": "‏yyyy-'ж'., d-MMM, EE",
-                "s": "‏d/M/yy, E"
+                "f": "yyyy-'ж'., d-MMMM, EEEE",
```

Comment:
I don't see what the difference is here. Even when I click "hide whitespace differences", the changes in this file keep showing up...

Reply:
The difference is `200f` rtl mark which is removed. When the tool generating data, if it's rtl, it added `200f` in front which is removed now in this data.

Reply:
I see... that char is not visible in the git diffs and it is there before this used to be RTL language. Now with the switch to Cyrillic, it is not considered RTL any more.